config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml

```diff
@@ -0,0 +1,336 @@
+        - --env-file=jobs/env/ci-kubernetes-kubemark-gce-big.env
+        - --extract=local
+        - --gcp-master-size=n1-standard-4
+        - --gcp-node-size=n1-standard-8
+        - --gcp-nodes=7
+        - --gcp-project=k8s-presubmit-scale
+        - --gcp-zone=us-east1-b
+        - --kubemark
+        - --kubemark-nodes=500
+        - --provider=gce
+        - --stage=gs://kubernetes-release-pull/ci/pull-kubernetes-kubemark-e2e-gce-big
+        - --tear-down-previous
+        - --test=false
+        - --test_args=--ginkgo.focus=\[Feature:Performance\] --gather-resource-usage=true --gather-metrics-at-teardown=true
+        - --timeout=100m
+        # TODO(krzyzacy): Figure out bazel built kubemark image
+        #env:
+        # - name: KUBEMARK_BAZEL_BUILD
+        # value: "y"
+        # Bazel needs privileged mode in order to sandbox builds.
+        securityContext:
+          privileged: true
+        resources:
+          requests:
+            memory: "6Gi"
+
+  - name: pull-kubernetes-kubemark-e2e-gce-big
+    always_run: true
+    skip_report: false
+    max_concurrency: 12
+    branches:
+    - release-1.10  # per-release image
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+      preset-bazel-scratch-dir: "true"
+      preset-dind-enabled: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20180730-8b7ab3104-1.10
+        args:
+        - --root=/go/src
+        - "--job=$(JOB_NAME)"
+        - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+        - "--repo=k8s.io/release"
+        - "--service-account=/etc/service-account/service-account.json"
+        - "--upload=gs://kubernetes-jenkins/pr-logs"
+        - "--timeout=120"
+        - --scenario=kubernetes_e2e
+        - --
+        - --build=bazel
+        - --cluster=
+        - --env-file=jobs/env/ci-kubernetes-e2e-kubemark-common.env
+        - --env-file=jobs/env/ci-kubernetes-kubemark-gce-big.env
+        - --extract=local
+        - --gcp-master-size=n1-standard-4
+        - --gcp-node-size=n1-standard-8
+        - --gcp-nodes=7
+        - --gcp-project=k8s-presubmit-scale
+        - --gcp-zone=us-east1-b
+        - --kubemark
+        - --kubemark-nodes=500
+        - --provider=gce
+        - --stage=gs://kubernetes-release-pull/ci/pull-kubernetes-kubemark-e2e-gce-big
+        - --tear-down-previous
+        - --test=false
+        - --test_args=--ginkgo.focus=\[Feature:Performance\] --gather-resource-usage=true --gather-metrics-at-teardown=true
+        - --timeout=100m
+        # TODO(krzyzacy): Figure out bazel built kubemark image
+        #env:
+        # - name: KUBEMARK_BAZEL_BUILD
+        # value: "y"
+        # Bazel needs privileged mode in order to sandbox builds.
+        securityContext:
+          privileged: true
+        resources:
+          requests:
+            memory: "6Gi"
+
+  - name: pull-kubernetes-kubemark-e2e-gce-big
```

Comment:
I think you can remove this job since 1.9 is too old for us to care running this presubmit on.

Reply:
huh? this is the previous kubemark job right? why we don't want to guard 1.9, 1.10 branches? Those branches are still actively maintained.

Reply:
You're right. However, given that older branches get cherrypicks (which originate from master branch and hence are already tested), it should be ok. Those jobs consume quite some resources and I don't think there is much value running against those.

Reply:
If we really want it later, we can re-add those branches. In any case, I don't think there is much point having multiple copies of the job config. Wdyt?

Reply:
sure, up to you